app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskViewModel.kt

```diff
@@ -96,6 +96,7 @@ class AddEditTaskViewModel(
     }
 
     private fun onDataNotAvailable() {
+        // TODO should this show some sort of error to differentiate it from a new load?
```

Comment:
That's a feature request for master

Reply:
Right, I'll make an issue about showing errors in general for when data can't be loaded